Plugins/GitUIPluginInterfaces/GitUIPluginInterfaces.csproj

```diff
@@ -60,6 +60,9 @@
     <DocumentationFile>bin\Release\GitUIPluginInterfaces.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="JetBrains.Annotations, Version=11.1.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\JetBrains.Annotations.11.1.0\lib\net20\JetBrains.Annotations.dll</HintPath>
+    </Reference>
```

Comment:
Added a reference to `JetBrains.Annotations`. This assembly is only used at design time and the compiler won't copy it to the output folder. The attributes themselves are conditional.

In future I'll make another PR that removes annotations from the project's source code and uses this package throughout.